### PR TITLE
Components

### PR DESCRIPTION
**Pull request by HunterX-7**
## Dear Code reviewer In this Project, I:

- Followed Gitflow.
- Use React Linters.
- Write meaningful commit messages in the most important parts of the process.
- Add the following features to the project:
    > Develop just the UI of the calculator
    > Use class-based components
    > Import it to my main component (App.js)

image

## I Hope you have a great day and your support is much appreciated! 😄

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,11 @@
...
 import './App.css';
+import Calculator from './components/calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
```
Comment (KDlamini) [inline src/App.js:10]: Please make sure that the `App.js` component is class-based. As much as it is better to use functional components and you will later refactor your code. The point of the curriculum is to learn how to use both class-based and functional components. For now, let's use class-based components until further notice.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @HunterX-7 🙋‍♀️,  

Excellent work! 💯 
You have done a great job creating components for the Math Magicians React application 🎊 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Good PR description
✔️  Well-documented README
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HunterX-7** [comment]
## Project changes

The following changes were implemented as requested:

- Please make sure that the App.js component is class-based.

**Review by christianbiring1 — APPROVED** [review]
Hi @HunterX-7 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
imageswelljob

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `summary` :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**